openfair.db

```diff
@@ -0,0 +1 @@
+../ofdb-inhalt/openfair.db.normal
```

Comment:
please remove this file!

Reply:
... btw: you could add `openfair.db` to `.gitignore` :)

Reply:
done